css/styles.css

```diff
@@ -151,7 +151,7 @@ body {
     transform: translateY(-1px);
     -moz-box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
     -webkit-box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
-    box-shadow: 3px 2px 5px rgba(0, 0, 0, 0.2); }
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2); }
```

Comment:
Making the box shadow 0 for both top and bottom will make the border radius appears good on button.